bin/deploy-all-backer

```diff
@@ -20,6 +22,11 @@ if [[ "$ETHERSCAN_API_KEY" != "" ]]; then
     echo "Found ETHERSCAN_API_KEY in environment, will attempt to verify contracts."
 fi
 
+loadAddresses $ADDRESS_FILE
+if [[ -z "$GOD_ADDR" || -z "$CURRENCY" ]]; then
```

Comment:
I would require `NFT_COLLATERAL` as well. Technically it is not required for the deployment but if it is not included in the final JSON it won't work in the UI

Reply:
"Technically it is not required" - we will hopefully soon have a deployment that will have multiple. I strongly disagree that it should be included. UI and contracts should be separated.